float64 bin_x
float64 bin_y
string bin_stat